src/store/modules/data/crawler.js

```diff
@@ -17,9 +17,9 @@ function getInitialState() {
 }
 
 export const crawlerError = createAction(CRAWLER_ERROR)
-const startCrawler = createAction(CRAWLER_START)
+export const startCrawler = createAction(CRAWLER_START)
```

Comment:
I don't think we should export this, it does nothing on its own and should only be used internally in this module, same for finishCrawler